docs/usecases/datastorage_for_institutions.rst

```diff
@@ -157,162 +423,232 @@ about where data are stored, as they can access it just as easily as before.
-configured link to ``$DATA``: It is configured as a remote to clone and pull
-data from, but upon creation of the dataset, the dataset's directory is also created at the remote
-end as a bare repository to enable pushing of results back to ``$DATA``. At the same
-time, a GitLab :term:`sibling` in the institute's GitLab instance is created, with a
-publication dependency on the data storage.
+   link HCP usecase
+
+shows an example where git-annex key storage is explicitly not wanted. For most
+storage or back-up scenarios, special remote capabilities are useful, though,
+and thus the default [#f4]_.
+The :command:`datalad create-sibling-ria` command will automatically create a
+dataset representation in a RIA store (and set up the RIA store, if it does not
+exist), and configure a sibling to allow publishing to the RIA store and updating
+from it.
+With special remote capabilities enabled, the command will create a second
+sibling to the git-annex special remote. With these two siblings set up, upon an
+invocation of :command:`datalad publish --to <sibling> --transfer-data all`,
+the complete dataset contents, including annexed contents, will be published
+to the RIA store, with no further setup or configuration required [#f5]_.
+
+RIA store workflows
+"""""""""""""""""""
+
+.. index:: ! datalad command; create-sibling-ria
+
+A RIA store can be created or extended by running the :command:`datalad create-sibling-ria`
+command (:manpage:`datalad-create-sibling-ria` manual) in an existing dataset.
+Supply a sibling name of your choice with the ``-s/--sibling`` option, and specify
+the location of the store as a ``ria+`` URL. Beyond this, there are no further
+required arguments.
+
+.. findoutmore:: What about optional arguments?
+
+   - unless it is explicitly given via ``--ria-remote-name``, the sibling to the
+     ria-remote special remote will have the same sibling name suffixed with ``-ria``.
+   - Special remote capabilities of a RIA store can be disabled at the time of
+     RIA store creation by using the option ``--no-ria-remote``
+   - :term:`Permissions` in the RIA store can be configured via ``--shared``. It
+     will default to multi-user access, but can take any specification of
+     the :command:`git init --shared` argument (find out more in the
+     `documentation <https://git-scm.com/docs/git-init>`__ of the command).
+
+
+RIA stores can be used under different types of file transfer protocols.
+Depending on the file transfer protocol, the looks of the URL can differ:
+
+- :term:`SSH`: ``ria+ssh://[user@]hostname:/absolute/path/to/ria-store``
+- Local file system: ``ria+file:///absolute/path/to/ria-store``
+- :term:`http` (e.g., to a RIA store like `store.datalad.org <http://store.datalad.org/>`_):
+  ``ria+http://store.datalad.org:/absolute/path/to/ria-store``
+
+Note that it is required to specify an :term:`absolute path` in the URL. Here is
+how one could create two siblings, ``server_juseless`` and ``server_juseless-ria``,
```

Comment:
It might be confusion to think about them as two siblings, because they are actually the same. One is the git-remote, and one is the special remote pointing to the same storage.